build-and-test.ps1

```diff
@@ -22,12 +23,17 @@ $manifestRepo = $manifest.Repos[0]
 $activeOS = docker version -f "{{ .Server.Os }}"
 $builtTags = @()
 
+$buildFilter = $Filter
+if ($activeOS -eq "windows" -and $buildFilter -like "2.*") {
```

Comment:
`-eq` should we do a `-ieq` instead?

Reply:
fixed

Reply:
wait - what is the benefit of `-ieq` over `eq`?  Are they equivalent?